README.md

```diff
@@ -67,6 +69,11 @@ on Debian based distributions the `plugin` command is located in:
 
 `/usr/share/elasticsearch/bin/plugin`
 
+* Install Python modules
+```
+python setup.py install
```

Comment:
Good stuff, thank you @luminitacristinalazarescu !
One minor thing:
`python setup.py develop`
This makes sense at this stage since there's not a pypi release